pkg/provisioner/ironic/updateopts_test.go

```diff
@@ -905,11 +905,6 @@ func TestGetUpdateOptsForNodeLiveIsoToImage(t *testing.T) {
 			Path: "/instance_info/boot_iso",
 			Op:   nodes.RemoveOp,
 		},
-		{
-			Path:  "/deploy_interface",
-			Value: "direct",
-			Op:    nodes.AddOp,
-		},
```

Comment:
It would be nice if we set up the test so that we could verify that the Remove op is applied when necessary.